### PR TITLE
Bug 2115020: Removed sync interval update and create from mirror peer

### PR DESCRIPTION
**Pull request by GowthamShanmugam**

### CODE REVIEW

**Comment by SanjalKatiyar** [comment]
> /hold
mco opr PR: https://github.com/red-hat-storage/odf-multicluster-orchestrator/pull/150 needs to be merged before this change.

File: packages/mco/components/disaster-recovery/create-dr-policy/create-dr-policy.tsx [hunk 1]
```diff
@@ -57,10 +54,8 @@ const fetchMirrorPeer = (
     return existingPeerNames.sort().join(',') === peerNames.sort().join(',');
   });
 
-type ReRouteResourceProps = {
-  history: RouteComponentProps['history'];
-  match: Match<{ url: string }>;
-};
+const getDRPolicyLispPageLink = (url: string) =>
```
Comment (SanjalKatiyar) [inline packages/mco/components/disaster-recovery/create-dr-policy/create-dr-policy.tsx:57]:
```suggestion
const getDRPolicyListPageLink = (url: string) =>
```

Comment (GowthamShanmugam) [reply packages/mco/components/disaster-recovery/create-dr-policy/create-dr-policy.tsx:57]: ack

**Comment by SanjalKatiyar** [comment]
> > /hold
> > mco opr PR: [red-hat-storage/odf-multicluster-orchestrator#150](https://github.com/red-hat-storage/odf-multicluster-orchestrator/pull/150) needs to be merged before this change.

/hold cancel
mco PR is merged...

File: packages/mco/components/disaster-recovery/create-dr-policy/create-dr-policy.tsx [hunk 2]
```diff
@@ -133,92 +128,69 @@ export const CreateDRPolicy: React.FC<ReRouteResourceProps> = ({
...
+        kind: MirrorPeerModel.kind,
+        metadata: { generateName: 'mirror-peer-' },
+        spec: {
+          manageS3: true,
+          type: state.replication,
+          items: state.selectedClusters.map((cluster) => ({
+            clusterName: cluster?.name,
+            storageClusterRef: {
+              name: cluster.storageClusterName,
+              namespace: CEPH_STORAGE_NAMESPACE,
```
Comment (vbnrh) [inline packages/mco/components/disaster-recovery/create-dr-policy/create-dr-policy.tsx:170]: Are we assuming this to be constant ? Should probably be fetched from via clusterClaim be possible

File: packages/mco/components/disaster-recovery/create-dr-policy/create-dr-policy.tsx [hunk 3]
```diff
@@ -133,92 +128,69 @@ export const CreateDRPolicy: React.FC<ReRouteResourceProps> = ({
...
-                  ...mirrorPeer?.spec?.schedulingIntervals,
-                  state.syncTime,
-                ]),
-              ],
+    if (Object.keys(mirrorPeer).length === 0) {
+      // MirrorPeer creation
+      const mirrorPeerPayload: MirrorPeerKind = {
+        apiVersion: getAPIVersionForModel(MirrorPeerModel),
+        kind: MirrorPeerModel.kind,
+        metadata: { generateName: 'mirror-peer-' },
```
Comment (vbnrh) [inline packages/mco/components/disaster-recovery/create-dr-policy/create-dr-policy.tsx:162]:
```suggestion
        metadata: { generateName: 'mirrorpeer-' },
```

nit

**Review by vbnrh — CHANGES_REQUESTED** [review]

**Comment by vbnrh** [comment]
Should be followed up by adding a cluster claim in ocs operator for the storage cluster namespace.

**Comment by GowthamShanmugam** [comment]
> Should be followed up by adding a cluster claim in ocs operator for the storage cluster namespace.

ack

**Pull request merged**